src/page-modules/assistant/trip/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { formatLocaleTime, secondsToDuration } from '@atb/utils/date';
```

Comment:
I renamed the `trip-pattern` folder to `trip` because I thought the `Trip` and `TripPattern` components could co-exist here